### PR TITLE
Rectify some mistakes from #170 in ActivityListCommand

### PR DESCRIPTION
**Pull request by lelerer**

### CODE REVIEW

File: src/main/java/seedu/splitlah/command/ActivityListCommand.java [hunk 1]
```diff
@@ -33,16 +33,16 @@ public ActivityListCommand(int sessionId) {
...
 
             if (activityListToBePrinted.isEmpty()) {
                 manager.getUi().printlnMessage(Message.ERROR_ACTIVITYLIST_ACTIVITY_EMPTY);
                 return;
             }
 
             for (Activity activity : activityListToBePrinted) {
-                manager.getUi().printlnMessageWithDivider(activity.toString());
+                manager.getUi().printlnMessage(activity.getActivitySummaryString());
             }
```
Comment (froststein) [inline src/main/java/seedu/splitlah/command/ActivityListCommand.java:47]: Please remove the extra line here, likewise for line 68

**Review by froststein — CHANGES_REQUESTED** [review]
Remove extra spaces in lines of code.

**Review by froststein — APPROVED** [review]
Looks good to merge

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Pull request merged**